docs/services/frontend.md

```diff
@@ -1,10 +1,10 @@
-# frontend
+# Front End
```

Comment:
I'm not sure about this one, here are some opinions https://www.designernews.co/stories/42206-frontend-frontend-or-front-end-developer

Reply:
I'm going to leave it as frontend